### ISSUE TITLE
[Q] How to reproduce values sampled by Bayesian method?

### ISSUE THREAD

**Issue by scho97** [wandb/client#2766]
Hello,

I am using W&B Sweeps to optimize hyperparameters for my neural network. I noticed that when I run my code several times with random seed fixed, I get different results, and this seems to be caused by different hyperparameter values sampled by the Bayesian method.

```
optim_method = "bayes"
metric = {"name": "best_acc", "goal": "maximize"}
```

I was wondering if there is any way to seed the Bayesian sampler provided by W&B. Or is there any other way encouraged by Sweeps for reproducing the run?

Thank you.

**Comment by scho97** [comment]
Hi Leslie,

Thank you for your reply. Your suggestions make sense. I'll look into them.

All the best,
SungJun
